React/App.js

```diff
@@ -0,0 +1,267 @@
+import React, {Component} from 'react';
+import {Text, View} from 'react-native';
+import {Location, Permissions, MapView} from 'expo';
+
+// import Toast from 'react-native-smart-toast';
```

Comment:
While commenting out code in development is very useful. In production code comments should only be describing difficult to understand bits of code.
https://stackoverflow.com/questions/3664288/is-it-bad-practice-to-leave-commented-out-code-in-production-releases?utm_medium=organic&utm_source=google_rich_qa&utm_campaign=google_rich_qa